Loop/Managers/LoopDataManager.swift

```diff
@@ -182,6 +178,9 @@ final class LoopDataManager: ObservableObject {
         self.carbAbsorptionModel = carbAbsorptionModel
         self.usePositiveMomentumAndRCForManualBoluses = usePositiveMomentumAndRCForManualBoluses
         
+        self.publishedMostRecentGlucoseDataDate = mostRecentGlucoseDataDate
```

Comment:
I think this will be nil, as displayState.input is nil until a fetchData happens.  You could do `glucoseStore.latestGlucose?.startDate`